hibernate-reactive-quickstart/src/main/java/org/acme/hibernate/reactive/FruitMutinyResource.java

```diff
@@ -35,15 +34,12 @@
 @Produces("application/json")
 @Consumes("application/json")
 public class FruitMutinyResource {
-    private static final Logger LOGGER = Logger.getLogger(FruitMutinyResource.class.getName());
 
     @Inject
-    Mutiny.Session session;
+    Session session;
 
     @GET
     public Uni<List<Fruit>> get() {
-        // In this case, it makes sense to return a Uni<List<Fruit>> because we return a reasonable amount of results
-        // Consider returning a Multi<Fruit> for result streams
```

Comment:
Why did you remove this comment? It's there because people are puzzled by the fact that we are not using a `Multi`

Reply:
Because the comment is confusing and misleading: "in this case"?? What is special about this case? We've been through this many times: `Multi` doesn't make sense with today's RDBMSs. The comment makes it seem like it would sometimes make sense, which is simply wrong.

Reply:
Several people have found it weird that we don't return a `Multi<Fruit>` in this case. The case in question is returning the result from a query or more precisely when to use `Multi<Fruit>` vs `Uni<List<Fruit>>`.

We had several discussion about this online but I think a comment here to mention this makes sense.
Now, the previous comment wasn't great but I would replace it with something better instead of removing it completely.